bin/add_frontend_server.rb

```diff
@@ -220,5 +225,21 @@ def execute_ssh_on_channel(ssh, command, exit_error_string)
   ssh.exec!("rm /tmp/*#{file_suffix}*")
 end
 
-puts "To deploy new chef instance, run the following:"
-puts "ssh gateway -t knife bootstrap #{private_dns_name} -x ubuntu --sudo -E production -N #{instance_name} -r role[front-end]"
+case options['environment']
+when 'production'
+  cmd =  "ssh gateway -t knife bootstrap #{private_dns_name} -x ubuntu --sudo -E production -N #{instance_name} -r role[front-end]"
+when 'adhoc'
+  cmd =  "ssh gateway -t knife bootstrap #{private_dns_name} -x ubuntu --sudo -E adhoc -N #{instance_name} -r role[unmonitored-standalone]"
+else
```

Comment:
Good suggestion, this simplifies this code by getting rid of the case statement and makes it easy to add new environments (once the recipes are good)